src/Cones/Cones.jl

```diff
@@ -46,7 +46,7 @@ function dot_expr(
     vars::AbstractVecOrMat{<:Union{VR,AE}},
     opt::Optimizer,
 )
-    return JuMP.@expression(opt.oa_model, JuMP.dot(z, vars))
```

Comment:
Ouch! I don't think accessing functions from another module through `JuMP.` was part of the public API.

Reply:
Because JuMP uses `using LinearAlgebra` previously, so these functions are leaked.
```julia
julia> module A                       
       using LinearAlgebra            
       end                            
Main.A                                
                                      
julia> import .A                      
                                      
julia> A.dot                          
dot (generic function with 42 methods)
                                      
julia> A.dot([1,2], [3,4])            
11                                    
```

Reply:
Yeah, but they weren't exported, so they were private:
```julia
julia> import Pkg

julia> Pkg.status("JuMP")
      Status `/private/tmp/h1/Project.toml`
  [4076af6c] JuMP v1.0.0
  
julia> using JuMP

julia> dot
ERROR: UndefVarError: dot not defined

julia> JuMP.dot
dot (generic function with 78 methods)

julia> :dot in names(JuMP)
false
```

Interesting edge-case though. People will depend upon all behavior...

Reply:
Julia does not hide any internal function with `<module>.<function>` notation.
A good example of Hyrum's Law.